frontend/src/js/standard-query-editor/actions.ts

```diff
@@ -148,9 +148,13 @@ const useLoadBigMultiSelectValues = () => {
                     ...table,
                     filters: await Promise.all(
                       table.filters.map(async (filter) => {
-                        if (filter.type !== "BIG_MULTI_SELECT") return filter;
-                        if (!filter.value || filter.value.length === 0)
+                        if (
+                          filter.type !== "BIG_MULTI_SELECT" ||
+                          !filter.value ||
+                          filter.value.length === 0
```

Comment:
[Impliziert](https://developer.mozilla.org/en-US/docs/Glossary/Falsy?retiredLocale=de) `!filter.value` nicht auch `filter.value.length === 0`? Wenn ja könnte letztere bedingung weg

Reply:
No it doesn't. Check `if (![]) console.log("hello")` doesn't log anything

Reply:
Ah okay, ich war davon ausgegangen, dass `filter.value` immer ein String ist.